packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -214,9 +215,17 @@ contract L2Genesis is Deployer {
         // 1B,1C,1D,1E,1F: not used.
         setSchemaRegistry(); // 20
         setEAS(); // 21
+        setRevenueSharer(); // 24
         setGovernanceToken(); // 42: OP (not behind a proxy)
     }
 
+    function setRevenueSharer() public {
+        RevenueSharer(Predeploys.REVENUE_SHARER).initialize({
+            _beneficiary: cfg.revenueShareRecipient(),
+            _l1Wallet: cfg.revenueShareRemainderRecipient()
+        });
+    }
```

Comment:
you should be sure to call initialize on the implementation too, with null values